hugo/content/about.md

```diff
@@ -0,0 +1,58 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.
+
+## Source Code & Development
+
+The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see [Overview of Repositories on GitHub](https://github.com/mumble-voip/).
+
+## Licenses
+
+Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a BSD-style [License](https://www.mumble.info/LICENSE).
+
+All Licenses are also listed in the GitHub repositories, see: [Main Mumble License](https://github.com/mumble-voip/mumble/blob/master/LICENSE) & [Overview of other Mumble Projects](https://github.com/mumble-voip/)
+
+
+# About us
+
+We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.
+
+## Core Team
+
+Active Team Members are:
+
+| Name: | Github Profile: | Assignment: |
+| --- | --- | --- |
+| Davide Beatrici | [davidebeatrici](https://github.com/davidebeatrici) | Core developer |
+| Robert Adam | [Krzmbrzl](https://github.com/Krzmbrzl) | Core developer, GitHub Management |
+| Nik Reist | [ZeroAbility](https://github.com/ZeroAbility) | Release Management |
+| Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |
```

Comment:
I feel this should represents the current situation (that's also why Thorvald isn't listed at all).
Nonetheless you are free to suggest a different role description here. This was just my original suggestion and since you did not answer on my ping on this topic, we left it at that :shrug:

Reply:
@Krzmbrzl Could you maybe talk about this in one of your team meetings, so that we could have an accurate representation of every team member?
@Kissaki What would you like as a description?

Reply:
Whatever

Reply:
"Whatever" is not really a good answer. Maybe other team members want a different description etc. as well. I will wait for the results of your team meeting, thus will unresolve this.